storage/migrations/02_runtimes.up.sql

```diff
@@ -207,6 +207,19 @@ CREATE TABLE chain.evm_token_balance_analysis
 );
 CREATE INDEX ix_evm_token_balance_analysis_stale ON chain.evm_token_balance_analysis (runtime, token_address, account_address) WHERE last_download_round IS NULL OR last_mutate_round > last_download_round;
 
+CREATE TABLE chain.evm_contracts
+(
+  runtime runtime NOT NULL,
+  contract_address oasis_addr NOT NULL,
+  creation_tx HEX64 NOT NULL,
```

Comment:
IIUC we have to allow NULL until we have tracing

Reply:
Yeah, I think so as well. I don't think we'll be able to figure out the creation tx for contracts discovered via `evm.Code()`.